src/firmware_logger_device.cpp

```diff
@@ -0,0 +1,219 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "firmware_logger_device.h"
+#include "ds5/ds5-private.h"
+#include "l500/l500-private.h"
+#include "ivcam/sr300.h"
+#include <string>
+
+namespace librealsense
+{
+    std::map<firmware_logger_device::logs_commands_group, firmware_logger_device::logs_commands>
+        firmware_logger_device::_logs_commands_per_group =
+    {
+        {LOGS_COMMANDS_GROUP_DS, {command{ds::GLD, 0x1f4}, command{ds::FRB, 0x17a000, 0x3f8}}},
```

Comment:
Why centralize?

Reply:
I prefer this to remain as is, so that we can see all the commands in the same place.